pkg/rt/manager.go

```diff
@@ -149,3 +175,10 @@ type interrupt struct {
 	cancel func()
 	done   <-chan error
 }
+
+func parseKey(key string) (string, string) {
+	ks := strings.Split(key, sep)
+	return ks[0], ks[1]
```

Comment:
strings.Split does not guarantee that it's return value will have cardinality 2, so you can't access indices 0 and 1 without a check.